spec/systems/keywords/view_keyword_detail_spec.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe 'View keyword details', type: :system do
+  include ActiveJob::TestHelper
+
+  # rubocop:disable RSpec/ExampleLength
+  it 'displays a link to keywords#show for succeeded keyword and displays the pagination nav' do
+    keyword = Fabricate(:parsed_keyword)
+
+    visit(keyword_path(keyword))
+
+    expect(page).to(have_selector('p[data-testid="keyword_details_content"]'))
+    expect(page).to(have_selector('p[data-testid="keyword_details_links_count"]'))
+    expect(page).to(have_selector('p[data-testid="keyword_details_top_ads_count"]'))
+    expect(page).to(have_selector('p[data-testid="keyword_details_total_ads_count"]'))
+    expect(page).to(have_selector('p[data-testid="keyword_details_non_ads_count"]'))
+    expect(page).to(have_selector('p[data-testid="keyword_details_ads_urls"]'))
+    expect(page).to(have_selector('p[data-testid="keyword_details_non_ads_urls"]'))
+    expect(page).to(have_selector('div[data-testid="keyword_details_page_view"]'))
```

Comment:
Only checking for having the selectors is not sufficient. The test would still pass if the scraped data was empty. We must ensure that the scraped content is displayed, i.e., checking the text value in each DOM selector.

Reply:
Test case improved in 9be9a361e317cafab5bb7ab0eb04e3b714f1581f